applications/add_file_to_db.py

```diff
@@ -0,0 +1,145 @@
+        If no path is given, the file is assumed to be in the CWD.
+    directory (str, required if fileName isn't given)
+        A directory with files to upload to the DB. \
+        All files in the directory with a predefined list of extensions will be uploaded.
+    db (str)
+        The DB to insert the files to. \
+        The choices are either the default CTA simulation DB or a sandbox for testing.
+    verbosity (str, optional)
+        Log level to print (default=INFO).
+
+    Example
+    -------
+    uploading a dummy file.
+
+    .. code-block:: console
+
+        python applications/add_file_to_db.py -f data/data-to-upload/test-data.dat
+'''
+
+import logging
+import argparse
+from pathlib import Path
+
+from simtools import db_handler
+import simtools.util.general as gen
+
+
+def userConfirm():
+    '''
+    Ask the user to enter y or n (case-insensitive).
+
+    Returns
+    -------
+    bool: True if the answer is Y/y.
+    '''
+
+    answer = ''
+    while answer not in ['y', 'n']:
+        answer = input('Is this OK? [y/n]').lower()
+
+    return answer == 'y'
+
+
+if __name__ == '__main__':
+
+    db = db_handler.DatabaseHandler()
+
+    parser = argparse.ArgumentParser(
+        description=(
+            'Add a file or files to the DB.'
+        )
+    )
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument(
+        '-f',
+        '--fileName',
+        help=(
+            'The file name to upload. '
+            'A list of files is also allowed, in which case only one -f is necessary, '
+            'i.e., python applications/add_file_to_db.py -f file_1.dat file_2.dat file_3.dat '
+            'If no path is given, the file is assumed to be in the CWD.'
+        ),
+        type=str,
+        nargs='+',
+    )
+    group.add_argument(
+        '-d',
+        '--directory',
+        help=(
+            'A directory with files to upload to the DB. '
+            'All files in the directory with the following extensions '
+            'will be uploaded: {}'.format(', '.join(db.ALLOWED_FILE_EXTENSIONS))
+        ),
+        type=str,
+    )
+    parser.add_argument(
+        '-db',
+        dest='dbToInsertTo',
+        type=str,
+        default=db.DB_TABULATED_DATA,
+        choices=['sandbox', db.DB_TABULATED_DATA],
+        help=(
+            'The DB to insert the files to. '
+            'The choices are {0} or "sandbox", '
+            'the default is {0}'.format(db.DB_TABULATED_DATA)
+        )
+    )
+    parser.add_argument(
+        '-v',
+        '--verbosity',
+        dest='logLevel',
+        action='store',
+        default='info',
+        help='Log level to print (default is INFO)'
+    )
+
+    args = parser.parse_args()
+
+    logger = logging.getLogger()
+    logger.setLevel(gen.getLogLevelFromUser(args.logLevel))
+
+    if args.fileName is not None:
+        filesToInsert = args.fileName
```

Comment:
What about we check for the allowed extension in case of a single file?

Reply:
Good idea, will add this check now.